src/routes/resources/resources.handlers.ts

```
@@ -149,6 +174,28 @@ export const patch: AppRouteHandler<PatchRoute> = async (c) => {
+    ? existingResource.image
+    : resource.image?.name + nanoid(5);
+  if (resource.image) {
+    if (existingResource.image) {
+      await c.env.MY_BUCKET.delete(existingResource.image);
+    }
+    const imageR2 = await c.env.MY_BUCKET.put(newImageKey, resource.image!);
+    if (!imageR2) {
+      return c.json(
+        { message: "Failed to upload image", success: false },
+        HttpStatusCodes.INTERNAL_SERVER_ERROR
+      );
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)